### DISCUSSION TITLE
Elegoo Neptune 4 Pro Moonraker

### DISCUSSION THREAD

**Discussion by nickyboy1979** [Arksine/moonraker#910] [General]
Heeeellllp!
I deleted some of the older versions of cura (may not even be related) and now moonraker is not connecting, ive uninstalled and re installed but no joy, if i open the fluuid bookmark in my browser it sees the printer but if i try to send from Cura i get a connection refused error.
When upload to printer i get "NetworkError.ConnectionRefusedError - Connection refused" in Cura. when I try to access the Moonraker web UI I get "This site can’t be reached 192.168.1.196 refused to connect."
The Fluidd UI is connecting, everything was working last night and the only thing i have done was delete the old curas which may not even be a part of the problem but its the only thing I changed
Heres the moonraker log SS from this morning
many thnks
![Screenshot 2024-10-08 121817032212312](https://github.com/user-attachments/assets/761fb1c4-f54f-49be-b643-5eab8f635e8a)

**Comment by Pyrolite1** [comment]
were you able to fix your issue? I am having a similar issue with moonraker on my Neptune 4 plus that i received for Christmas and dont know how to fix it.
